### DISCUSSION TITLE
Wrap comments

### DISCUSSION THREAD

**Discussion by zilleste** [biomejs/biome#4447] [Ideas]
Not sure if this is out of scope for Biome, but it would be neat if it could automatically rewrap text in multi-line comments, in accordance with lineWidth.

This has also been a long-standing Prettier feature request: https://github.com/prettier/prettier/issues/265

**Comment by arendjr** [comment]
I will admit, I would personally love this feature.

I am somewhat of a mind to say we could implement this without breaking Prettier compatibility, since Prettier doesn’t change the comments, so any change we would make is respected and formatted the same by Prettier.

I know, maybe that reasoning is a bit backwards, but I’m just trying to think how we can justify having this feature :)

**Reply by izakfilmalter** [reply]
Would love this!

**Comment by elieven** [comment]
Bump++

**Comment by Svish** [comment]
Please, please add this! Manually line-breaking multi-line comments is _so_ annoying!
